### PR TITLE
Scaled dog images

### PR DESCRIPTION
**Pull request by ShmulTomer**
## Dog Images Scaling

image

image

Issue Number(s): #153

What does this PR change and why?
- scaled dog images on the **Individual Dog Page** and **Dog Card Dashboard View**

Question:
- this is probably addressed with the individual dog page mobile view, but how should the size of the image container itself scale as the screen becomes smaller?

### Checklist

- [ ] Requirements have been implemented
- [ ] Acceptance criteria is met
- [ ] Database schema [docs](https://www.notion.so/gtbitsofgood/Database-Schema-Docs-fd377ded10b54890b65fd99025488cee?pvs=4) have been updated or are not necessary
- [ ] Relevant reviewers (EM) have been assigned to this PR

### Critical Changes

None

### How To Test

The steps to test the functionality of your ticket. Include edge cases and testing of error handling, if applicable.

### CODE REVIEW

File: src/components/Dog/CardDogTable.jsx [hunk 1]
```diff
@@ -35,7 +35,16 @@ function DogCard({ className, dog, onClick }) {
       onClick={onClick || (() => {})}
     >
       <div className="flex w-[240px] h-[240px] items-center justify-center rounded-lg bg-primary-gray overflow-hidden">
-        {dog.image ? <Image alt={dog.name} width={350} height={350} src={dog.image} />
+        {dog.image ?
```
Comment (afazio1) [inline src/components/Dog/CardDogTable.jsx:37]: is there a reason you are using regular styles instead of tailwind? I would prefer you to use tailwind to keep consistency throughout the codebase

**Review by afazio1 — CHANGES_REQUESTED** [review]

**Comment by dograh** [comment]
Design looks good!

**Review by afazio1 — APPROVED** [review]
